DEVELOP.md

```diff
@@ -313,6 +299,12 @@ local clone of the ``pywbem/pywbem`` Git repo.
 
         .. _`list of open issues`: https://github.com/pywbem/pywbem/issues
 
+4.  Add an initial Git tag for the new release stream and push it to the
+    remote repo:
+
+        git tag ${MNU}a0
```

Comment:
I would guess that the a0 above needs explanation.

Reply:
Agreed. Added "This is necessary because the version is calculated based on the most recent tag in the commit history."